index.html

```diff
@@ -899,8 +919,11 @@ <h2>Specifying the Type</h2>
 <!--
 {
   ####...####
-  "@id": "http://example.org/places#BrewEats",
-  "****@type****": ****[ "http://schema.org/Restaurant", "http://schema.org/Brewery" ]****####,
+  "@id": "http://me.markus-lanthaler.com/",
+  "@type": ****[****"
+     http://schema.org/Person",
```

Comment:
Whitespace broke the data(base) star. 😉 

Travis was mostly right...it's Example 14 though, not 13 per Travis:
https://pr-preview.s3.amazonaws.com/w3c/json-ld-syntax/pull/47.html#x3-4-specifying-the-type

Reply:
Matching example numbering in an external script is challenging, and I'll need to look into that more later.

Reply:
Actually, numbering seems to be off because of the example in typographical conventions, which isn't considered by the script, as it's in an included file. It may just be an off-by-one problem, but changing that to use Example 0 might make sense anyway.